CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [D] Changes is `Cere Dev` Runtime
 
 ## [VNext]
+- [C,D] `pallet-ddc-clusters`: Added Erasure coding and Replication in cluster params
```

Comment:
Please, put this change record under the `[5.3.0]` section, we deprecated the `[VNext]` section